app/views/welcome/index.html.erb

```diff
@@ -1,3 +1,8 @@
+<%if user_signed_in?%>
+  <div class="position-absolute top-50 start-50 translate-middle">
```

Comment:
- [ ] I see that your live demo link isn't working. Please deploy your app on Netlify or Heroku and update the live demo link in the README file. 🙏 

_Please notice that the corresponding file for this requirement isn't loaded in your PR, so I had to leave the comment in a random file._

Reply:
bro, i have problems with netlify, and heroku too

Reply:
This issue is not yet resolved.

Reply:
bro i have a problem when i try to do the deploy on heroku

Reply:
i'm so exhausted now and I am trying to find a different ways to deploy my project, I promise you that I'm gonna looking for the alternative